generators/server/templates/src/main/java/package/repository/_UserRepository.java

```diff
@@ -109,6 +109,9 @@
     @Cacheable(cacheNames = "users")
     <%_ } _%>
     Optional<User> findOneWithAuthoritiesByLogin(String login);
+
+    @EntityGraph(attributePaths = "authorities")
```

Comment:
Why this request is not `@Cacheable` as opposed to `findOneWithAuthoritiesByLogin`?
Especially if this new method is the first called, the cache could be necessary.

Reply:
@hdurix could be done, but this cache needs to be maintained in the `UserService` as well.

Reply:
Yes you're right, for eviction

Reply:
@hdurix added caching for `findOneWithAuthoritiesByLogin`